hivemq-edge/src/frontend/src/locales/en/components.json

```
@@ -130,12 +130,31 @@
           "ariaLabel": "List of subscriptions to be created",
           "noDataText": "There is no rows to display"
         },
+        "columns": {
+          "admin": "Admin columns",
+          "subscription": "Subscription columns",
+          "selectHeader": "Select all rows",
+          "selectRow": "Select the row"
+        },
         "action": {
           "delete_zero": "Delete rows",
           "delete_one": "Delete {{ count }} row",
           "delete_other": "Delete {{ count }} rows",
           "errorOnly": "Show only errors"
         }
+      },
+      "confirm": {
+        "alert": {
+          "title_zero": "No valid subscriptions",
+          "title_one": "One subscription created",
+          "title_other": "{{ count}} subscriptions created",
```

question(non-blocking): not sure if the syntax will fail or not, just raising awareness here

```suggestion
          "title_other": "{{ count }} subscriptions created",
```